app/src/main/java/io/actifit/fitnesstracker/actifitfitnesstracker/MainActivity.java

```diff
@@ -3013,7 +3265,7 @@ public String getFormattedValue(float value, Entry entry, int dataSetIndex, View
 
                 //add limit lines to show marker of min 5K activity
                 //YAxis yAxis = chart.getAxisLeft();
-                dayBarData.setValueFormatter(yFormatter);
+                dayBarData.setValueFormatter((ValueFormatter) yFormatter);

```

Comment:
why aren't you using yformatter defined and initialized above?

Reply:
It was showing an error